tests/integration_generic.test.ts

```diff
@@ -28,8 +28,10 @@ const CHAIN_NAME = "Kusama";
 const CHAIN_TICKER = "ksm";
 const YOUR_PUBKEY = "d280b24dface41f31006e5a2783971fc5a66c862dd7d08f97603d2902b75e47a";
 const YOUR_ADDRESS = "HLKocKgeGjpXkGJU6VACtTYJK4ApTCfcGRw51E5jWntcsXv";
-const YOUR_BLOB =
-  "040000313233343536373839303132333435363738393031323334353637383930313233158139ae28a3dfaac5fe1560a5e9e05cd5038d2433158139ae28a3dfaac5fe1560a5e9e05c362400000c000000b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafeb0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe";
+const YOUR_CALL_DATA =
+  "0000d050f0c8c0a9706b7c0c4e439245a347627901c89d4791239533d1d2c961f1a72ad615c8530de078e565ba644b38b01bcad249e8c0";
+const YOUR_SIGNED_EXTENSIONS =
```

Comment:
Check if signed_extensions have to be divided in `include_in_extrinsic` and `included_in_signed_data`

Reply:
Solved with new approach!